http/src/main/resources/reference.conf

```diff
@@ -41,8 +41,9 @@ dataproc {
   monitor {
     initialDelay = 30 seconds
     pollingInterval = 15 seconds
-    checkToolsDelay = 8 seconds
```

Comment:
Previously `checkToolsDelay` was not used (the value was hardcoded in `BaseCloudServiceRuntimeMonitor`). I made it be used and also added `checkToolsMaxAttempts`.

Reply:
the original intent of `checkToolsDelay` was to have an initial wait before we ping jupyter or welder, but I removed it later since I felt it wasn't benefiting much, and there're cases where the tools might be ready when we call checkTools. I think you're using it as `interval`, shall we rename it to `checkToolsInterval` instead?

Reply:
Ah yeah I am using as initial delay and interval.. sound good renaming it to `checkToolsInterval`